Debug App/Sources/View Controllers/MerchantHeadlessCheckoutAvailablePaymentMethodsViewController.swift

```diff
@@ -170,17 +170,16 @@ extension MerchantHeadlessCheckoutAvailablePaymentMethodsViewController: UITable
         let paymentMethod = self.availablePaymentMethods[indexPath.row]
         let paymentMethodType = paymentMethod.paymentMethodType
         if paymentMethodType == "PAYMENT_CARD" ||
-            paymentMethodType == "ADYEN_BANCONTACT_CARD"
-        {
+            paymentMethodType == "ADYEN_BANCONTACT_CARD" {
             let alert = UIAlertController(title: "", message: "Select Implementation", preferredStyle: .actionSheet)
             
-            let rawDataAlertAction = UIAlertAction(title: "Raw Data", style: .default , handler:{ (UIAlertAction)in
+            let rawDataAlertAction = UIAlertAction(title: "Raw Data", style: .default, handler: { (_)in
```

Comment:
One for the future: there should be a rule for omitting parentheses for non-tuple arguments